app/models/work.rb

```diff
@@ -11,6 +11,7 @@ class Work < ApplicationRecord
      "permission_description",
      "location_of_composition",
      "author",
+     "recipient",
```

Comment:
This adds the `recipient` field to IIIF exports as the export uses the Work's `PUBLIC_ATTRIBUTES` to generate its metadata.